addons/t9n/views/t9n_resource_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="t9n.resource_view_form" model="ir.ui.view">
+        <field name="name">Resource</field>
+        <field name="model">t9n.resource</field>
+        <field name="arch" type="xml">
+            <form string="Resource Form">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="file_name" readonly="True"/>
+                        </group>
+                        <group>
+                            <field name="file" filename="file_name"/>
```

Comment:
It would be nice to add a filter to hide irrelevant stuff in the file explorer:
```suggestion
                            <field name="file" filename="file_name" options="{'accepted_file_extensions': '.po,.pot'}"/>
```